File: c++/1.1 Herencia Simple y Modificadores de acceso protegidos/1.1.2 Modificadores/PersonaV1Test.cpp
#include <iostream>

using namespace std;

class PersonaV1 {
  public: 
    string nombre;
    int edad;
    float estatura;

    string toString() {
      return "{nombre='" + nombre + "'" + 
            ", edad='" + std::to_string(edad) + "'" + 
            ", estatura='" + std::to_string(estatura) + "'" + 
            "}";
    }
};

int main(int argc, char const *argv[])
{
  PersonaV1 persona = PersonaV1();
  persona.nombre = "Fernando López";
  persona.edad = 22;
  persona.estatura = 1.70f;

  cout << persona.toString() << endl;
  // {nombre='Fernando López', edad='22', estatura='1.7'}
  
  return 0;
}

File: python/1.1 Herencia Simple y Modificadores de acceso protegidos/1.1.2 Modificadores/PersonaV1Test.py
class PersonaV1:
  nombre = ""
  edad = 0
  estatura = 0.0
  
  def __str__(self):
    return f"[nombre='{self.nombre}', edad='{self.edad}', estatura='{self.estatura}']"

# Uso de la clase Persona
if __name__ == "__main__":
  persona = PersonaV1();
  persona.nombre = "Fernando López";
  persona.edad = 22;
  persona.estatura = 1.7;
  
  print(persona);
  # [nombre='Fernando López', edad='22', estatura='1.7']